server/lib/handlebars.js

```diff
@@ -152,6 +152,36 @@ handlebars.registerHelper('encodeURIComponent', str => {
 
 handlebars.registerHelper('formatCurrencyObject', (obj, props) => formatCurrencyObject(obj, props.hash));
 
+handlebars.registerHelper('formatOrderAmountWithInterval', order => {
+  if (!order.currency || !order.totalAmount) {
+    return null;
+  }
+
+  const formattedAmount = formatCurrency(order.totalAmount, order.currency);
+  const subscription = order.subscription;
+  let interval = null;
+
+  if (subscription !== undefined && subscription.interval !== null) {
+    interval = subscription.interval;
+  } else if (order.interval !== undefined && order.interval !== null) {
+    interval = order.interval;
+  }
+
+  if (interval !== null) {
+    let formattedAmountWithInterval = null;
+
+    if (interval === 'month') {
+      formattedAmountWithInterval = `(${formattedAmount}/m)`;
+    } else if (interval === 'year') {
+      formattedAmountWithInterval = `(${formattedAmount}/y)`;
+    }
+
+    return formattedAmountWithInterval;
```

Comment:
We can simplify here too

```suggestion
    if (interval === 'month') {
      return `(${formattedAmount}/m)`;
    } else if (interval === 'year') {
      return `(${formattedAmount}/y)`;
    }
```